clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -411,8 +411,8 @@ public int memberEpoch() {
     public void onHeartbeatSuccess(ConsumerGroupHeartbeatResponseData response) {
         if (response.errorCode() != Errors.NONE.code()) {
             String errorMessage = String.format(
-                    "Unexpected error in Heartbeat response. Expected no error, but received: %s",
```

Comment:
nit: White space change in a file you aren't otherwise changing, I'd avoid it.

Reply:
good call.  I think it was editor's auto correction.  Reverting it.